app/views/courses/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>New Course</h1>
```

Comment:
I know this is boilerplate but we are 100% not going to be keeping this so I would just remove this entire file.